webview-ui/src/i18n/locales/ja/chat.json

```diff
@@ -1,5 +1,5 @@
 {
-	"greeting": "Rooにお手伝いできることはありますか？",
+	"greeting": "ROOへようこそ",
```

Comment:
Let's preserve "Roo Code" instead of "Roo" or "ROO" when we refer to the name of the product. The agent can refer to itself as "Roo" though.